apps/xprof_core/test/xprof_core_test_lib.erl

```diff
@@ -15,6 +16,13 @@ is_elixir_version(Requirement) ->
             false
     end.
 
+-spec is_elixir_available() -> boolean().
+is_elixir_available() ->
+    case os:find_executable("elixir") of
```

Comment:
I don't think it is a good idea - what if Elixir is installed, but no Elixir code used inside that BEAM instance?

Reply:
Yeah, that's not a good idea, I've just found that out the hard way.

Reply:
I think we still need to do all steps
- check `elixir` executable is on OS path
- if yes ensure it can be loaded (`get_elixir_ebin` can still fail if elixir version is not compatible with erlang version)
- starting `elixir` application ensures loading of some Elixir code

(this allows us running just the ct tests without relying on eunit loading elixir)